brigade/static/css/style.css

```diff
@@ -308,3 +302,56 @@ tbody th {
   }
 
 }
+
+.free-software__logo {
+  margin: 40px 0;
+}
+
+.inkind-card-list {
+  display: flex;
+  flex-wrap: wrap;
+  margin-bottom: 50px;
+}
+
+a.inkind-card {
+  border-radius: 10px;
+  border: 3px solid #5a5a5a;
+  box-shadow: 2px 2px 2px rgba(0, 0, 0, 0.2);
+  color: #5a5a5a; /* TODO: extract this as a text color variable */
+  flex-shrink: 0;
+  font-weight: initial;
+  margin: 0 20px 20px 0;
+  padding: 10px;
+  text-align: center;
```

Comment:
Centered paragraph text is little harder to read than left-aligned text.
![image](https://user-images.githubusercontent.com/8628090/34124487-eecd39e6-e3e7-11e7-81f7-a87b58941d5e.png)
I would suggest having a ``.card__header`` and ``.card__body`` for each card, moving  each logo into the header and each descriptions into the body, and centering the header text and left-aligning the body text.
What do you think?

Reply:
I like that idea, but I wasn't able to get the left aligned text to be readable when I was trying it out. Also, I have a feeling that these cards are a bit of a special case (i.e. how many other cards are we going to need a giant logo to be the header?) That's where a bit of my hesitation to make this a reusable component just yet comes from.